src/index.html

```diff
@@ -7,8 +7,34 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;0,900;1,400;1,500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <nav class="nav">
+        <ul class="list_of_nav_links">
+          <li><a href="#" class="nav_link is-active">apple</a></li>
```

Comment:
```suggestion
          <li>
            <a href="#" class="nav_link is-active">
              apple
            </a>
          </li>
```

Reply:
move children to new lines to make your code more readable and don't forget about indentation 
Fix everywhere please